content/docs/0.17.x/reference/cli/commands/keptn_configure_bridge.md

```diff
@@ -35,7 +35,7 @@ keptn configure bridge --user=<user> --password=<password>
       --config-file string   Specify custom Keptn Config file path (default: ~/.keptn/config)
   -h, --help                 help
       --mock                 Disables communication to a Keptn endpoint
-  -n, --namespace string     Specify the namespace where Keptn should be installed, used and uninstalled in (default "keptn")
+  -n, --namespace string     Specify the namespace where Keptn should be installed, used and uninstalled (default "keptn")
```

Comment:
This sounds like the configure bridge command is going to install Keptn -- is that true?  Or should this say "... where Keptn is installed" or "...where the Keptn Helm chart is installed"?